ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/validation/dcat/blanknodevalidators/DcatBlankCatalogNode.java

```diff
@@ -0,0 +1,23 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.validation.dcat.blanknodevalidators;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.vocabulary.RDF;
+
+import java.util.List;
+
+import static be.vlaanderen.informatievlaanderen.ldes.server.domain.validation.dcat.DcatValidator.DCAT_CATALOG;
+
+public class DcatBlankCatalogNode implements DcatBlankNodeValidator {
```

Comment:
DcatBlankCatalogNode is in terms of naming a bit inconsistent -> DcatBlankCatalogNodeValidator